open-backend/routes/message_routes.js

```diff
@@ -1,17 +1,18 @@
 const express = require('express');
-var router = express.Router();
-const message_controller = require('../controllers/message_controller');
+const router = express.Router();
+const {message_write_post, message_reply_post, message_read, message_list} = require('../controllers/message_controller');
```

Comment:
Stylistically, using const message_controller = require(...) is better here, since if we added new functions to the message controller, we wouldn't have to worry about adding them to this require statement as well